src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -302,6 +303,20 @@ class RsErrorAnnotator : AnnotatorBase(), HighlightRangeExtension {
         error.addToHolder(holder)
     }
 
+    private fun checkUnstableAttribute(ref: RsReferenceElement, o: RsElement, holder: RsAnnotationHolder) {
+        if (o.containingCargoPackage?.origin == PackageOrigin.STDLIB) return
+        val element = ref.reference.resolve() as? RsOuterAttributeOwner ?: return
+        for (attr in element.queryAttributes.unstableAttributes) {
+            val metaItems = attr.metaItemArgs?.metaItemList ?: continue
+            val featureName = metaItems.singleOrNull { it.name == "feature" }?.value ?: continue
+            val reason = metaItems.singleOrNull { it.name == "reason" }?.value
+            val reasonSuffix = if (reason != null) ": $reason" else ""
+            val feature = CompilerFeature.find(featureName)
+                ?: CompilerFeature(featureName, FeatureState.ACTIVE, null, cache = false)
+            feature.check(holder, o, null, "`$featureName` is unstable$reasonSuffix")
```

Comment:
Let's highlight item more precisely - only the corresponding identifier instead of the whole path/dot expression.

| Actual | Expected |
|-|-|
| <img width="342" alt="Screen Shot 2019-12-09 at 16 32 48" src="https://user-images.githubusercontent.com/2539310/70439670-90d29c80-1aa1-11ea-9c86-65a38a36cdff.png"> | <img width="361" alt="Screen Shot 2019-12-09 at 16 29 58" src="https://user-images.githubusercontent.com/2539310/70439646-84e6da80-1aa1-11ea-88f4-40bb9a372263.png"> |

Don't forget to add tests to check compound paths